platform/jewel/foundation/src/main/kotlin/org/jetbrains/jewel/foundation/lazy/SelectableLazyColumn.kt

```diff
@@ -122,6 +151,90 @@ public fun SelectableLazyColumn(
         }
     }
 
+    val notifyingPointerEventActions =
```

Comment:
Can you add some local documentation to this with a high-level explanation of why you added this wrapper? Just to keep some references in the code explaining the emission.